intro-HPC/ch_account.tex

```diff
@@ -382,6 +406,9 @@ \section{First Time connection to the \hpc}
       \ifgent
       Make sure the fingerprint in the Alert matches \strong{2f:0c:f7:76:87:57:f7:5d:2d:7b:d1:a1:e1:86:19:f3}
       \fi
+      \ifbrussel
+      Make sure the fingerprint in the Alert matches \strong{41:68:9b:c2:f0:1c:d3:62:14:b6:71:f0:36:8a:7d:ec}
```

Comment:
As well as I was not aware with many other changes. Maybe I have to delete this PR.